game_api.txt

```diff
@@ -313,6 +313,27 @@ Allows creation of new fences with "fencelike" drawtype.
 	sounds = default.node_sound_wood_defaults(),
 
 
+Torch API
+---------
+
+Allows creation of new torches with "wallmounted" drawtype.
```

Comment:
'wallmounted' is not a drawtype, the drawtype of these torches is 'mesh'. 'wallmounted' is a rotation type.
I think it is better to describe this in a simpler way as 3D torches:
"Allows creation of new 3D torches."